lib/ansible/modules/network/eos/eos_config.py

```diff
@@ -44,6 +44,8 @@
         in the device running-config.  Be sure to note the configuration
         command syntax as some commands are automatically modified by the
         device config parser.
+      - Abbreviated commands in lines are NOT idempotent, see
+        L(Network FAQ,../network/user_guide/faq.html).
```

Comment:
Can we link directly to the article?, rather than the top of the FAQ page?

Reply:
Good idea. I don't know, but I'll give it a try.